rosdep/python.yaml

```
@@ -1798,6 +1798,15 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python3-flask-sqlalchemy:
+  arch: [python-flask-sqlalchemy]
+  debian: [python3-flask-sqlalchemy]
+  fedora: [python3-flask-sqlalchemy]
+  gentoo: [dev-python/flask-sqlalchemy]
+  nixos: [python39Packages.flask_sqlalchemy]
+  ubuntu:
+    '*': [python3-flask-sqlalchemy]
```

Sorry, I just noticed this:

```suggestion
  ubuntu: [python3-flask-sqlalchemy]
```